services/kuber/server/server.go

```diff
@@ -31,6 +32,11 @@ const (
 	outputDir        = "services/kuber/server/clusters"
 )
 
+type ClusterMetadata struct {
+	NodeIps    map[string][]net.IP `json:"node_ips"`
```

Comment:
What are the key and values here? From the code it looks like `[nodepool-name]<public ips in nodepool>[]`, but I would expect it to be something like `[node-name]<public IP, private IP>[]`, or something along those lines.

It would be easier for a user to find IP for individual node, not to look for it in nodepool.

Reply:
Yes currently it's [nodepool-name][ array of all public ips for each node]
`[node-name]<public IP, private IP>[]` works as well 👍 , I think it might be better.

Reply:
the data will now look like this

```json
{
    "node_ips": {
        "matus-t1-qa47cb5-hetzner-compute-1": {
            "public_ip": "192.168.2.2",
            "private_ip": "192.168.2.2"
        },
        "matus-t1-qa47cb5-hetzner-control-1": {
            "public_ip": "192.168.2.1",
            "private_ip": "192.168.2.1"
        }
    },
    "private_key": "...."
}
```